### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by karayamanemre**
In this PR:
- Created a new Rails app.
- Installed needed node dependencies as described in the previous lesson's tutorial.
- Created a babel.config.js in the root folder of the project as described in the previous lesson's tutorial.
- Updated webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
- Project has a Postgres database set up.
- Created a table for storing messages and created 5 different greetings.
- Created an API endpoint that selects a random greeting from the table.
- Created a static view as a root.
- Created the App component with react-router and rendered it in the static view.
- Created the Greeting component that will display a greeting.
- Created a store, an action, and a reducer.
- Displayed the random greeting in the Greeting component.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Emre, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe is professionally documented, and adhere(s) to the following standard(s):
- [ ]  The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later. 
- [ ] Kindly ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. Similarly, kindly also remove guide notes for _license_ creation under the _license_ section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by karayamanemre** [comment]
Really? @rloterh

**Review by Whoistolu — COMMENTED** [review]
Hi @karayamanemre 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have its README professionally documented.
- [x] You did a great just Licensing the app properly 👍🏽 🥇 
- [x] As recommended by the previous Reviewer, kindly add at least 2 potential features you think this application will have in the future. This will take care of the Future Features section.
- [x] Also, let's ensure that the Acknowledgment section is not empty. Putting a dot there does not look professional enough. Kindly give credit to everyone who inspired your codebase.
- [x] I can also see that you have 1,776 file changes that may not allow a reviewer to properly review your project. Please, take a look at your `node_modules` directory and ensure that it is not been pushed to Github.
### File Changes
Screenshot 2023-02-20 at 09 31 22


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "Hello, world!",
+  "Selamin Aleyküm",
+  "Naptın müdür",
+  "OOOOO Naber Yaa!",
+  "Welcome aboard!"
+]
+
+greetings.each do |message|
+  Greeting.create!(message: message)
+end
```
Comment (Whoistolu) [inline db/seeds.rb:18]: - Great implementation @karayamanemre 👍🏽 
Screenshot 2023-02-20 at 10 46 34

**Review by Whoistolu — APPROVED** [review]
### Hi @karayamanemre 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

### Highlight
✔️ `jsbundling-rails` dem was used
✔️ `react-dom`, dependencies `babel-loader` are present in the `package.json` file
✔️ API endpoint for returning a random greeting was created
✔️ Database table for storing different greetings was created
✔️ There is a `Greeting` component
✔️ The two apps work together

- [ ] [OPTIONAL] Great job implementing my previous recommendations, However, you need to update the `Acknowledgment` and `Fututure-Feautre` sections with the appropriate details. In the Acknowledgment section, Kindly give credit to everyone who inspired your codebase. In the Future-Feature section, kindly add at least 2 potential features you think this application will have in the future.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**